dmoj/executors/mixins.py

```diff
@@ -114,6 +114,19 @@ def get_env(self):
                 return env
 
             def launch(self, *args, **kwargs):
+                if 'symlinks' in kwargs:
+                    for src, dst in kwargs['symlinks'].items():
```

Comment:
Why not `for src, dst in kwargs.get('symlinks', {}).items()`?

Reply:
I think this code is more explicit in what it's actually doing. Note that on all *our* invocations of `launch`, symlinks will be an empty dict due to the default config entry.

Reply:
I fail to see how the fact it's always on our invocations makes any difference. I just think it fits better with the rest of the file to use `.get` instead of `in` and `__getitem__`, as well as reduce the amount of indentation.